test/clj/rems/api/test_applications.clj

```diff
@@ -749,6 +749,61 @@
                   "application.command/approve"]
                  (:application/permissions (get-application-for-user app-id decider)))))))))
 
+(deftest test-promote-to-applicant
+  (let [applicant "alice"
+        handler "developer"
+        app-id (test-helpers/create-application! {:actor applicant})]
+    (testing "submit application and add two members"
+      (is (= {:success true}
+             (send-command applicant {:type :application.command/submit
```

Comment:
Could introduce the `ok-command` variant of `send-command` that does the comparison?

Reply:
Hmm, good suggestion. One argument against is that with an `(is (= ` you get slightly nicer errors (like "expected `{:success true}` but was `{:success false :errors [{:type :forbidden}]}`)

Reply:
Hmm how do we manage in the other places with this?

Reply:
This file uses the `(is (= {:success true} ...)` a lot

Reply:
Potentially there could be some machinery for assertions or exception messages here like `assert-response-is-ok` or so but I don't think it makes this much better. Usually there are other fields to check so an `is map` is fine. It seems to be this file that does this a lot but I guess not enough to matter.

Reply:
Also not worth a special macro :)